### PR TITLE
Really handle lib64

### LINKED ISSUE #605: option always-copy doesn't work with OOTB CentOS 6.5
Using the latest release of CentOS (6.5)
Install virtualenv as root:

```
yum install python-setuptools
easy_install pip
pip install virtualenv
```

As a regular user try to use --always-copy option with virtualenv:

``````
$ virtualenv --always-copy foo
Cannot find file lib (bad symlink)
New python executable in foo/bin/python
Installing setuptools, pip...
  Complete output from command /home/frede00e/foo/bin/python -c "import sys, pip; sys...d\"] + sys.argv[1:]))" setuptools pip:
  Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python2.6/site-packages/virtualenv_support/pip-1.5.5-py2.py3-none-any.whl/pip/__init__.py", line 9, in <module>
  File "/usr/lib/python2.6/site-packages/virtualenv_support/pip-1.5.5-py2.py3-none-any.whl/pip/log.py", line 6, in <module>
  File "/usr/lib64/python2.6/logging/__init__.py", line 34, in <module>
    import sys, os, types, time, string, cStringIO, traceback
ImportError: No module named time
----------------------------------------
...Installing setuptools, pip...done.
Traceback (most recent call last):
  File "/usr/bin/virtualenv", line 9, in <module>
    load_entry_point('virtualenv==1.11.5', 'console_scripts', 'virtualenv')()
  File "/usr/lib/python2.6/site-packages/virtualenv.py", line 824, in main
    symlink=options.symlink)
  File "/usr/lib/python2.6/site-packages/virtualenv.py", line 992, in create_environment
    install_wheel(to_install, py_executable, search_dirs)
  File "/usr/lib/python2.6/site-packages/virtualenv.py", line 960, in install_wheel
    'PIP_NO_INDEX': '1'
  File "/usr/lib/python2.6/site-packages/virtualenv.py", line 902, in call_subprocess
    % (cmd_desc, proc.returncode))
OSError: Command /home/frede00e/foo/bin/python -c "import sys, pip; sys...d\"] + sys.argv[1:]))" setuptools pip failed with error code 1```
``````

### PR DESCRIPTION
**Pull request by opoplawski**
Another attempt at this.

I believe this fixes #605, #530 #565 (which I believe are all duplicates).

Not sure about code style - I was trying to not break too much api if possible, but not sure how possible that is.

### CODE REVIEW

**Comment by opoplawski** [comment]
Ping?  I've found this essential to creating pyvenv rpms.

File: virtualenv.py [hunk 1]
```diff
@@ -1037,6 +1042,7 @@ def path_locations(home_dir):
         bin_dir = join(home_dir, 'bin')
     elif not is_win:
         lib_dir = join(home_dir, 'lib', py_version)
+        lib64_dir = join(home_dir, 'lib64', py_version)
```
Comment (Ivoz) [inline virtualenv.py]: Is this common convention across all major nix distros? And/or OS X?

**Comment by vitalyisaev2** [comment]
Need this feature a lot

**Comment by BrownTruck** [comment]
Hello!

As part of an effort to ease the contribution process and adopt a more standard workflow pip has switched to doing development on the ``master`` branch. However, this Pull Request was made against the ``develop`` branch so it will need to be resubmitted against ``master``. Unfortunately, this pull request does not cleanly merge against the current ``master`` branch.

If you do nothing, this Pull Request will be automatically closed by @BrownTruck since it cannot be merged.

If this pull request is still valid, please rebase it against ``master`` (or merge ``master`` into it) and resubmit it against the ``master`` branch, closing and referencing the original Pull Request.

If you choose to rebase/merge and resubmit this Pull Request, here is an example message that you can copy and paste:

```
Another attempt at this.

I believe this fixes #605, #530 #565 (which I believe are all duplicates).

Not sure about code style - I was trying to not break too much api if possible, but not sure how possible that is.

---

*This was migrated from pypa/virtualenv#757 to reparent it to the ``master`` branch. Please see original pull request for any previous discussion.*
```

**Comment by BrownTruck** [comment]
This Pull Request was closed because it cannot be automatically reparented to the ``master`` branch and it appears to have bit rotted.

Please feel free to re-open it or re-submit it if it is still valid and you have rebased it onto ``master`` or merged ``master`` into it.

**Pull request closed without merging**